vizro-core/changelog.d/20231011_110304_huong_li_nguyen_fix_css_bugs.md

```diff
@@ -0,0 +1,40 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+
+### Added
+
+- Add template for waterfall charts and default styling for `textarea` ([#106](https://github.com/mckinsey/vizro/pull/106))
```

Comment:
Should we split this into two separate notes?

Reply:
Yes, of course :)